lib/modem_info/modem_info.c

```diff
@@ -522,8 +522,8 @@ int modem_info_string_get(enum modem_info info, char *buf,
 		return err;
 	}
 
-	if (modem_data[info]->data_type == AT_PARAM_TYPE_NUM_SHORT) {
-		err = at_params_short_get(&m_param_list,
+	if (modem_data[info]->data_type == AT_PARAM_TYPE_NUM_INT) {
+		err = at_params_unsigned_short_get(&m_param_list,
```

Comment:
Please fix the alignment.

I see that the `param_value` is `uint16_t` but I wonder shall we limit the range of the variable in this case? The function is not bound to any particular parameter, it's just asked to return the integer as a string, IMO `at_params_int_get()` should be used and then make a string of whatever is returned. But not pushing if you don't want to change to be consistent with current (even if invalid) behaviour.

Reply:
Well, it's all based on L465:	uint16_t param_value;